src/main/java/com/justcode/company/Company.java

```diff
@@ -0,0 +1,37 @@
+package com.justcode.company;
+
+import com.justcode.company.employee.Employee;
+import com.justcode.company.exceptions.NotFoundAnyEmployeeException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Company {
+
+    private String name;
+    private List<Employee> employees;
+
+    public Company(String name) {
+        this.name = name;
+        this.employees = new ArrayList<>();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public List<Employee> getEmployees() throws NotFoundAnyEmployeeException {
```

Comment:
throws NotFoundAnyEmployeeException is redundant here

Reply:
Do you want I use way like this:
  try{
            throw new MyException(); 
        } 
        catch (MyException ex) 
        { 
            System.out.println("Caught"); 
            System.out.println(ex.getMessage()); 
        }